hack/lib/build/images.sh

```diff
@@ -127,12 +127,14 @@ function os::build::image::internal::docker() {
 	local extra_tag="${3-}"
 	local options=()
 
-	if ! docker build ${OS_BUILD_IMAGE_ARGS:-} -t "${tag}" "${directory}"; then
-		return "$?"
+	docker build ${OS_BUILD_IMAGE_ARGS:-} -t "${tag}" "${directory}"
```

Comment:
Because we're running with `set -o errexit` on this will change the control flow. The bug is real, though -- correct fix IMO is just to `return 1` in the `if`.

Reply:
yeah, that should work as well. I thought the actual return code was useful so I wanted to forward it as was probably originally intended.

Reply:
Sorry, I mis-remembered. The `set -o errexit` is _not_ on, so we need `if` to force the return. The return code not important, so `return 1` should be fine. See commit f661ac9d1cac752d19c6c251cfa02a7b630b7e98 in Origin.